### ISSUE TITLE
Cannot download JAR from maven using the README guide

### ISSUE THREAD

**Issue by marektoman** [swagger-api/swagger-codegen#10019]
##### Description

The Prerequisites guide in README.md doesn't work. 

When using wget, I get:
```
 501 HTTPS Required
```
After changing it to https:
```
ERROR: no certificate subject alternative name matches
        requested host name 'central.maven.org'.
To connect to central.maven.org insecurely, use `--no-check-certificate'.
```
After adding the switch:
```
403 Use https://repo1.maven.org
```
After changing the url:
```
404 Not Found
```
I've tried other variations to no avail.

The above is true for any version. More importantly, I had no success finding the 3.0.16 jar on any other maven repository.

**Comment by marektoman** [comment]
Found it:
```
iwr -out swagger-codegen-cli.jar https://repo1.maven.org/maven2/io/swagger/codegen/v3/swagger-codegen-cli/3.0.16/swagger-codegen-cli-3.0.16.jar
```

**Comment by frantuma** [comment]
Thanks for reporting this, fixed in #10021 and #10022
